packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/telephony-line-phone-accessories.controller.js

```diff
@@ -0,0 +1,104 @@
+import assign from 'lodash/assign';
+
+export default class TelephonyLinePhoneAccessoriesCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    atInternet,
+    OvhApiTelephony,
+    TelephonyMediator,
+    TucTelephonyAccessoriesOrderProcess,
+    TucToast,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.TelephonyMediator = TelephonyMediator;
+    this.TucTelephonyAccessoriesOrderProcess = TucTelephonyAccessoriesOrderProcess;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.process = null;
+    this.loading = {
+      init: false,
+    };
+    this.line = null;
+
+    this.loading.init = true;
+    this.TelephonyMediator.getGroup(this.billingAccount)
+      .then((group) => {
+        this.line = group.getLine(this.serviceName);
+      })
+      .then(() =>
+        this.OvhApiTelephony.Line()
+          .v6()
+          .get({
+            billingAccount: this.line.billingAccount,
+            serviceName: this.line.serviceName,
+          })
+          .$promise.then((result) => {
+            assign(
+              this.line,
+              { getPublicOffer: result.getPublicOffer },
+              {
+                isAttachedToOtherLinesPhone: result.isAttachedToOtherLinesPhone,
+              },
+            );
+          }),
+      )
+      .then(() => this.line.hasPendingOfferTasks())
+      .then(() => {
+        return this.line.getPhone();
+      })
```

Comment:
You can use the return shortcut
```suggestion
      .then(() => this.line.getPhone())
```